vector/src/main/java/im/vector/app/features/command/Command.kt

```diff
@@ -28,22 +28,29 @@ enum class Command(val command: String, val parameters: String, @StringRes val d
     EMOTE("/me", "<message>", R.string.command_description_emote),
     BAN_USER("/ban", "<user-id> [reason]", R.string.command_description_ban_user),
     UNBAN_USER("/unban", "<user-id> [reason]", R.string.command_description_unban_user),
+    IGNORE_USER("/ignore", "<user-id> [reason]", R.string.command_description_ignore_user),
+    UNIGNORE_USER("/unignore", "<user-id>", R.string.command_description_unignore_user),
     SET_USER_POWER_LEVEL("/op", "<user-id> [<power-level>]", R.string.command_description_op_user),
     RESET_USER_POWER_LEVEL("/deop", "<user-id>", R.string.command_description_deop_user),
+    ROOM_NAME("/roomname", "<user-id> [reason]", R.string.command_description_room_name),
```

Comment:
I know this PR is still a draft, but the command param `"<user-id> [reason]"` is not correct here (bad copy paste :))

Reply:
Oh